### PR TITLE
Use the stable image tags on the main branch

### PR DESCRIPTION
**Pull request by hdefazio**
Use the stable image tags

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: config/base/params.env [hunk 1]
```diff
@@ -1,6 +1,6 @@
-odh-model-controller=quay.io/opendatahub/odh-model-controller:v0.12.0
-caikit-tgis-image=quay.io/opendatahub/caikit-tgis-serving:stable-2d9db23
-caikit-standalone-image=quay.io/opendatahub/caikit-nlp:stable-994ec60
-tgis-image=quay.io/opendatahub/text-generation-inference:stable-eba83ba
-ovms-image=quay.io/opendatahub/openvino_model_server:2024.3-release-4c8c52c	
-vllm-image=quay.io/opendatahub/vllm:stable-849f0f5
```
Comment (spolti) [inline config/base/params.env:6]: shouldn't we keep the old ones instead just stable?

**Comment by hdefazio** [comment]
The failing image tag check is addressed in #316

File: config/base/params.env [hunk 2]
```diff
@@ -1,6 +1,6 @@
-odh-model-controller=quay.io/opendatahub/odh-model-controller:v0.12.0
+odh-model-controller=quay.io/opendatahub/odh-model-controller:stable
 caikit-tgis-image=quay.io/opendatahub/caikit-tgis-serving:stable-2d9db23
```
Comment (vaibhavjainwiz) [inline config/base/params.env:2]: Why we are commiting stable tag in main branch?
stable tag are created from release branch. Fast tags are created from main branch.

Comment (spolti) [reply config/base/params.env:2]: we have changed the main branch to stable and the incubating branch will be the new branch that receives PRs.

**Review by israel-hdez — APPROVED** [review]
/lgtm

**Comment by israel-hdez** [comment]
@hdefazio Since this PR is targeting `main` branch, you need to cherry-pick your PR #316 to `main` to fix CI.

**Comment by Jooho** [comment]
I sent a pr(https://github.com/opendatahub-io/odh-model-controller/pull/323) and I believe it will solve the main branch gitaction issue.

**Comment by Jooho** [comment]
my bad, my pr can not solve this issue. I think we can merge it manually

**Pull request merged**